codec2/msdk-codec2/mediacodec.te

```diff
@@ -0,0 +1,2 @@
+allow mediacodec proc_uptime:file { read write open };
```

Comment:
why also add these rules in graphics/mesa/mediacodec.te?

Reply:
It is needed to grant access to capture the crash messages.